Source/C++/Core/Ap4Dac4Atom.cpp

```diff
@@ -664,11 +664,12 @@ AP4_Dac4Atom::Ac4Dsi::SubStream::ParseSubStreamInfoAjoc(AP4_BitReader &bits,
         // oamd_common_data()
         ParseOamdCommonData(bits);
     }
-    n_umx_objects_minus1 = bits.ReadBits(4);
-    int nFullbandUpmixSignals = n_umx_objects_minus1 + 1;
+    int nFullbandUpmixSignalsMinus = bits.ReadBits(4);
```

Comment:
Can you explain what the intent of this change is? It seems that it does exactly the same as before, just with a new local variable.

Reply:
AC-4 level 4 supports up to 20.1 indicating the **n_umx_objects_minus1** could be 21. the old calculation only read 4 bits, didn't add 3 variable bits to **n_umx_objects_minus1**, the max number of **n_umx_objects_minus1** is 15. New calculation adds the additional 3 bits.